demo (1)/demo/src/main/java/com/ifour/EmployeeManagement/Department/DepartmentService.java

```diff
@@ -0,0 +1,55 @@
+package com.ifour.EmployeeManagement.Department;
+
+import com.ifour.EmployeeManagement.Employee.Employee;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Objects;
+import java.util.Optional;
+
+@Service
+public class DepartmentService {
+    @Autowired
+    public static DepartmentRepository departmentRepository;
+
+
+    public void updateDepartment(int dept_id, String dept_name) {
+        Department department = departmentRepository.findById(dept_id).orElseThrow(()->new IllegalStateException("Department with Id"+dept_id+"is not present"));
+
+        if(!Objects.equals(department.getDept_id(),dept_id))
+        {
+            department.setDept_id(dept_id);
+        }
+

```

Comment:
Didn't quiet understand the object comparison. We should talk about this in our next meeting.